service/worker/migration/activities.go

```diff
@@ -590,9 +590,17 @@ func (a *activities) verifyReplicationTasks(
 	details *replicationTasksHeartbeatDetails,
 	remoteClient adminservice.AdminServiceClient,
 	ns *namespace.Namespace,
+	heartbeat func(details replicationTasksHeartbeatDetails),
```

Comment:
Why do you need to pass in a func v.s. calling activity.heartbeat directly in the method?

Reply:
To make writing unit test easy. if I call `activity.heartbeat` directly, then I need to execute `verifyReplicationTasks` as an activity (at least I haven't another way to do it).